I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 49, in script
    jump choice1_m
ScriptError: could not find label 'choice1_m'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 49, in script
    jump choice1_m
  File "/home/protodrew/Documents/renpy-7.3.5-sdk/renpy/ast.py", line 1697, in execute
    rv = renpy.game.script.lookup(target)
  File "/home/protodrew/Documents/renpy-7.3.5-sdk/renpy/script.py", line 898, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'choice1_m'.

Linux-5.4.0-7642-generic-x86_64-with-debian-bullseye-sid
Ren'Py 7.3.5.606
A Quarter Past 5 1.0
Sun Sep 20 09:21:19 2020
